templates/coredns/schema.capnp

```diff
@@ -0,0 +1,34 @@
+@0xaf3b04a861b5a7f1;
+
+struct Schema {
+    etcdServerName @0: Text; #Name of Etcd service
+    upsteram @1 : Text; #by default 8.8.8.8:53 8.8.4.4:53
+    nodePort @2: Int32=9500; # the port to bind to
```

Comment:
should this be there|? cause it's not present in the documentation of the service

Reply:
and in all cases the default is 53 not 9500 but yea this doesn't need to be here. Either you get the port from the sal, or in the case of dns it will always be 53.